module-4/backend/week_one.md

```diff
@@ -0,0 +1,32 @@
+## Week One - Module 4 Recap
+
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+
+Note: When you're done, submit a PR. 
+
+1. What's the most useful thing you learned from completing the intermission week work?
+It was all useful, i can't think of one specific thing.
+2. What are some tools to help debug JavaScript code?
+pry and console.log()
+3. What are some tools you need in order to unit test your JavaScript?
+mocha / chai
+4. What is the syntax for invoking a function?
+function()
+5. What's the difference between `==` and `===` in JavaScript?
+'==' is converts both values to the same type and then compares, '===' does not do any type conversion
+6. What's the difference between asynchronous and synchronous JavaScript? 
+synchronous javascript runs sequentially and asynchronous calls a function before the previous function is finished
+7. How do we setup a route when creating an API with Node and Express?
+in the server file app.get or app.post then the route
+8. What do we use Knex for?
+Running/Controlling migrations
```

Comment:
It's also an ORM for interacting with SQL and database queries. It's similar to what?